tests/e2e/util.go

```diff
@@ -822,6 +825,17 @@ func newTestResourceQuota(name string, size string, scName string) *v1.ResourceQ
 	}
 }
 
+// existingResourceQuota returns a quota that enforces default constraints for testing
+func existingTestResourceQuota(name string, size string, namespace string) *v1.ResourceQuota {
```

Comment:
`updatedSpec4ExistingResourceQuota` might be better name
also what is the difference between this method and `newTestResourceQuota` can we combine them?
you don't seem to be using `namespace` param here

Reply:
In `newTestResourceQuota` we are creating resource quota for storage class, but it `existingTestResourceQuota` we are creating resource of type `requests.storage` it will be better if we are not combining those methods

Reply:
I have updated the function name  `existingTestResourceQuota` to `updatedSpec4ExistingResourceQuota`